docs/update.md

```diff
@@ -11,52 +11,36 @@ Distribution for PostgreSQL.
     Only the incremental update to a nearest minor version of the Operator is
     supported. To update to a newer version, which differs from the current
     version by more than one, make several incremental updates sequentially.
+    See [documentation archive](https://docs.percona.com/legacy-documentation/)
+    for documentation on previous versions of the Operator.
+
+Upgrading the Operator is similar to deploying a new Operator version, but you
+should change the `DEPLOY_ACTION` option in the `deploy/operator.yaml` file
+before applying it from `install` to `update`:
+
+```yaml hl_lines="7 8"
+...
+  containers:
+    - name: pgo-deploy
+      image: percona/percona-postgresql-operator:1.4.0-pgo-deployer
+      imagePullPolicy: Always
+      env:
+        - name: DEPLOY_ACTION
+          value: update
+...
+```
+
+You can automate this with the [yq tool](https://github.com/mikefarah/yq/#install) as
+follows, assuming that you are upgrading to the Operator version {{ release }}:
+
+``` {.bash data-prompt="$" }
+$  curl -s https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/operator.yaml | | yq w - "spec.template.spec.containers[0].env[3].value" "update" | kubectl apply -f -
+$ kubectl wait --for=condition=Complete job/pgo-deploy --timeout=90s
```

Comment:
@fiowro did you test the commnad? I see at least problem with` | | `
and we have only one env here https://github.com/percona/percona-postgresql-operator/blob/1.x/deploy/operator.yaml#L173